src/main/scala/org/constellation/p2p/Download.scala

```diff
@@ -348,26 +348,26 @@ object Download extends StrictLogging {
                     .flatMap(
                       _ =>
                         dao.cluster
-                          .compareAndSet(NodeState.validDuringDownload, oldState)
+                          .compareAndSet(NodeState.validDuringDownload, NodeState.Ready)
```

Comment:
I think it should stay as it was: `oldState`. It's error handler so in case that download failed, it shouldn't change state to ready then

Reply:
Fair point